### DISCUSSION TITLE
Still Waiting for Copilot Access as a Student

### DISCUSSION THREAD

**Discussion by dayaneglsantos** [community/community#145241] [GitHub Education]
### Hi everyone,

Hi, everyone!

On November 14th, I requested access to GitHub Copilot as a student, but I haven’t heard back yet. I know tools like this can be super helpful for us as we learn and work on projects.

Is anyone else experiencing delays in getting access? Any tips on what to do while I wait?

I’d appreciate it if you could share your experiences!

**Comment by davevad93** [comment]
Hi @dayaneglsantos ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @dayaneglsantos was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by dayaneglsantos** [comment]
Oh, that's great.
Thanks for explaining 👏🏻

**Reply by davevad93** [reply]
You're welcome @dayaneglsantos , I noticed that you now have a `PRO` badge, so you should already have access to your benefits! 👍

**Comment by dayaneglsantos** [comment]
Amazing!!!
Thanks again @davevad93
